.sequelizers

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+
+module.exports = {
```

Comment:
Why do you still have a `module.exports` ? Aren't you supposed to be using `ES6`.